astropy/units/astrophys.py

```diff
@@ -70,15 +70,16 @@
          doc="Electron mass",
          format={'latex': r'M_{e}', 'unicode': 'Mₑ'})
 # Unified atomic mass unit
-def_unit(['u', 'Da', 'Dalton'], 1.6605387e-27 * si.kg, namespace=_ns,
+def_unit(['u', 'Da', 'Dalton'], _si.u.value * si.kg, namespace=_ns,
          prefixes=True, exclude_prefixes=['a', 'da'],
          doc="Unified atomic mass unit")
 
 ##########################################################################
 # ENERGY
 
-def_unit(['Ry', 'rydberg'], 13.605692 * si.eV, namespace=_ns,
-         prefixes=True,
+def_unit(['Ry', 'rydberg'],
+         _si.Ryd.value * _si.c.value * _si.h.value / _si.e.value * si.eV,
+         namespace=_ns, prefixes=True,
```

Comment:
You have to be careful to pick the 2010 CODATA value also for Rydberg in eV, since we only have 2010 values for Rydberg (in 1/m) and planck constant in astropy (I assume the speed of light hasn't changed!). For [those] (http://www.physics.nist.gov/cuu/Constants/allascii_2010.txt),
```
Rydberg constant times hc in eV                             13.605 692 53            0.000 000 30             eV
```
i.e., the calculation is in fact consistent.

Now I do *really* hope that  the 2014 values are internally consistent as well.... we'll have to check in #4109.